jcl/src/java.management/share/classes/com/ibm/java/lang/management/internal/LoggingMXBeanImpl.java

```diff
@@ -83,10 +85,25 @@
 			Module java_logging = ModuleLayer.boot().findModule("java.logging").get(); //$NON-NLS-1$
 /*[ELSE]
 			Module java_logging = Layer.boot().findModule("java.logging").get(); //$NON-NLS-1$
-/*[ENDIF]*/			
-			Class<?> logManagerClass = Class.forName(java_logging, "java.util.logging.LogManager"); //$NON-NLS-1$
-			Class<?> loggerClass = Class.forName(java_logging, "java.util.logging.Logger"); //$NON-NLS-1$
-			Class<?> levelClass = Class.forName(java_logging, "java.util.logging.Level"); //$NON-NLS-1$
+/*[ENDIF]*/
+			Class<?> logManagerClass = AccessController.doPrivileged(new PrivilegedAction<Class<?>>() {
```

Comment:
Can these be written as lambdas?  Or backed by a single class?

Reply:
Updated the PR to invoke `PrivilegedAction` with `lambda` expression. In addition, stored three `Class.forName()` into an array to avoid creating three new classes as per Pete's comment. Please review again.